### PR TITLE
Add 'source_id' in ticket

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVP-1277

### CODE REVIEW

**Review by AurelienLP — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py [hunk 1]
```diff
@@ -92,6 +92,9 @@ class TicketSerializer(PbNestedSerializer):
     found = jsonschema.BoolField()
     cost = CostSerializer(display_none=True)
     links = jsonschema.MethodField(schema_type=LinkSchema(many=True))
+    source_id = jsonschema.Field(
+        schema_type=str, display_none=True, description='Product identifier of the object'
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py]:
```suggestion
        schema_type=str, display_none=True, description='Product identifier of the ticket'
```

**Review by kinnou02 — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]

**Review by nonifier — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py (hunk 1 above)
Comment (mbounabi) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py]: OK. Do I also change the other descriptions in TicketSerializer that say 'object' ?

Comment (kinnou02) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py]: you can yes, for the `id` you should indicate that it's the id of the ticket inside the response

Comment (mbounabi) [reply source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py]: Done... Except the part after the edit...

**Review by kadhikari — APPROVED** [review]

File: source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py [hunk 2]
```diff
@@ -86,12 +86,15 @@ def to_value(self, value):
...
-    id = jsonschema.Field(schema_type=str, display_none=True, description='Identifier of the object')
-    name = jsonschema.Field(schema_type=str, display_none=True, description='Name of the object')
+    id = jsonschema.Field(schema_type=str, display_none=True, description='Identifier of the ticket')
+    name = jsonschema.Field(schema_type=str, display_none=True, description='Name of the ticket')
     comment = jsonschema.Field(schema_type=str)
     found = jsonschema.BoolField()
     cost = CostSerializer(display_none=True)
     links = jsonschema.MethodField(schema_type=LinkSchema(many=True))
+    source_id = jsonschema.Field(
+        schema_type=str, display_none=True, description='Product identifier of the ticket'
```
Comment (pbougue) [inline source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py:96]: minor:
```suggestion
        schema_type=str, display_none=True, description='Product identifier of the ticket in the input data'
```

**Review by pbougue — APPROVED** [review]

**Review by kinnou02 — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

**Pull request merged**